cluster-api/cloud/google/config/configtemplate.go

```diff
@@ -147,6 +149,7 @@ spec:
         args:
         - --kubeconfig=/etc/kubernetes/admin.conf
         - --token={{ .Token }}
+        - --config=/etc/machinesetup/machine_setup_configs.yaml
```

Comment:
Call it this something other than `config` please. There are way too many things named `config`. Be specific here.